elastic/synthetics
# feat: move to playwright 1.26.0 version

+ fixes #609 
+ Fixes a test where the delay was not enough for capturing network data, just wanted to be stable instead of flaky. 

### Testing
+ Build the current PR
+ Install it globally
+ Use this synthetics version and build a HB package with these new playwright deps. 
+ Run browser monitors to make sure everything passes.
